ui/pages/Report/components/CustomSearch.jsx

```diff
@@ -22,7 +22,6 @@ import { getSearchIncludeAllProjectsInput, getSearchHashContextLoading } from '.
 const mapProjectsStateToProps = (state, ownProps) => ({
   project: getProjectsByGuid(state)[ownProps.value],
   projectSamples: getSamplesGroupedByProjectGuid(state)[ownProps.value],
-  loading: getProjectsIsLoading(state),
```

Comment:
the project detail endpoint has been completely deprecated by this PR, and this loading state was triggered by that endpoint, so it is now completely moot. When I went in to rework this, I realized it was unnecessary anyways because this was called for the filter so you can select which families in the project to search, but in the `CustomSearch` component that functionality is disabled (you can only select whether to include the whole project or not) so theres no need to fetch the families and therefore this will never fetch and never be loading